docs/tutorials/committing-liquidity.md

```diff
@@ -0,0 +1,143 @@
+
+:::info Read more
+[Building a bot tutorial](./building-a-bot/adding-a-liquidity-commitment.md): Basics on how to incorporate liquidity orders into an automated trading setup.
+
+[Rewards and penalties](../concepts/liquidity/rewards-penalties.md): LPs receive rewards (through fees paid by traders) when they meet the liquidity SLA. Fees are withheld, and they can be further penalised for not meeting their commitment.
+:::
+
+## Amending a liquidity commitment [WIP]
+When amending a liquidity commitment, the network will allow you to provide more bond for your liquidity immediately. However, if you reduce your liquidity commitment, it will only be enacted in the epoch after you submitted the amendment transaction.
+
+If you reduce your commitment to the point where the market would drop below its required [target stake](./../concepts/liquidity/provision.md#target-stake), then you may be penalised.
+
+### Transaction template
+
+Template to amend a liquidity commitment: 
+
+```
+submission = {
+    "liquidityProvisionAmendment": {
+        "marketId": market_id,
+        "commitmentAmount": "500000000000000000000",
+        "fee": "0.005",
+    },
+    "pubKey": pubkey,
+    "propagate": True
+}
+```
+
+## Cancelling a liquidity commitment [WIP]
+If you no longer want to keep a liquidity commitment open for a market, you can cancel your commitment using the `liquidityProvisionCancellation`. This will remove the commitment requirement from your public key, and return the bond amount back into your general account.
+
+If cancelling a commitment would leave a market without enough liquidity, then you will be penalised.
+
+Any open positions will not be cancelled.
+
+### Transaction template
+Template to cancel a liquidity commitment:
+
+```
+submission = {
+    "liquidityProvisionCancellation": {
+        "marketId": marketID
+    },
+    "pubKey": pubkey,
+    "propagate": True
+}
+```
+
+## Viewing existing liquidity provisions
+You can view the list of participants supplying liquidity to a market in two ways: using the endpoint or a Vega Tool. 
+
+**Use the endpoint**
+
+Querying the REST endpoint for a node given the partyID (public key) and marketID:
+`https://<node address>/liquidity-provisions/party/{party}/market/{market}`
+
+**Use the Vega Tool**
+
+Using the `vegatools` command line tool to view the liquidity providers for a given market. 
+
+[Vega Tools repo](https://github.com/vegaprotocol/vegatools): Try the liquidity commitment tool, under the `liquiditycommitment` folder. Check the **[README.md](https://github.com/vegaprotocol/vegatools#vegatools)** file for how to set up the Vega Tools.  
+
+To see the current liquidity commitments for a market on testnet, use this command line:
+
+```bash
+vegatools liquiditycommitment -a=n06.testnet.vega.xyz:3007
+```
+
+![Vega Tool for liquidity commitment](/img/tutorials/vegatools-liquidity-commitment.png)
+
+## What can go wrong when providing liquidity?
```

Comment:
I would put a link at the top of the page along the lines of "Providing liquidity is a significant and active commitment to providing orders on a Vega market. Failure to meet the commitment may result in penalties up to losing your entire bonded amount [link to this section]"